src/qmflows/packages/orca.py

```diff
@@ -70,11 +70,11 @@ def inithess(value: Any) -> Settings:
             # Convert Hessian to numpy array
             value = value if isinstance(value, np.ndarray) else np.array(value)
 
-            def format_atom(atom):
+            def format_atom(atom: plams.Atom) -> str:
                 symbol, mass, coords = atom.symbol, atom._getmass(), atom.coords
                 return '{:2s}{:12.4f}{:14.6f}{:14.6f}{:14.6f}\n'.format(symbol, mass, *coords)
 
-            def format_hessian(dim, hess):
+            def format_hessian(dim: int, hess: List[List[float]]) -> str:
```

Comment:
Just and small detail, hess could be a list of list or a numpy array

Reply:
Good point!
This does make annotating stuff a bit more tricky though, as the tools for annotating numpy arrays are... not exactly comprehensive.
An issue for the next pull request I suppose.